src/Identifier/Identifier.php

```diff
@@ -3,16 +3,27 @@
 
 namespace Spaze\PHPStan\Rules\Disallowed\Identifier;
 
+use PHPStan\BetterReflection\Reflector\Reflector;
+
 class Identifier
 {
+	private Reflector $reflector;
+
+
+	public function __construct(Reflector $reflector)
+	{
+		$this->reflector = $reflector;
+	}
+
 
 	/**
 	 * @param string $pattern
-	 * @param string $value
+	 * @param string|class-string $value
```

Comment:
Can this be a string only?

Reply:
Yeah, I think it helped with PHPStan but I've got the class_exists check so that should cover it.